galaxy.yml

```diff
@@ -39,6 +40,9 @@
         - install_galaxy
         - manage_postgres
 
+    - role: miniconda-role
+      tags: conda
+
```

Comment:
conda_auto_init seems to a bit fragile, so we use this role to install conda.
The last thing you see before conda gets installed is 
```
galaxy.tools.deps DEBUG 2016-09-16 15:18:05,018 Unable to find config file './dependency_resolvers_conf.xml'
PREFIX=/home/galaxy/tool_dependencies/_conda
installing: _cache-0.0-py27_x0 ...
installing: python-2.7.11-0 ...
installing: conda-env-2.4.5-py27_0 ...
installing: openssl-1.0.2g-0 ...
installing: pycosat-0.6.1-py27_0 ...
installing: pyyaml-3.11-py27_1 ...
installing: readline-6.2-2 ...
installing: requests-2.9.1-py27_0 ...
installing: sqlite-3.9.2-0 ...
installing: tk-8.5.18-0 ...
installing: yaml-0.1.6-0 ...
installing: zlib-1.2.8-0 ...
installing: conda-4.0.5-py27_0 ...
installing: pycrypto-2.6.1-py27_0 ...
installing: pip-8.1.1-py27_1 ...
installing: wheel-0.29.0-py27_0 ...
installing: setuptools-20.3-py27_0 ...
```

If for some reason this doesn't finish, there is a conda.lock file on the file system that needs to be removed first. We should fix this in galaxy to use a better mechanism.